docs/Acceptable-Formulae.md

```diff
@@ -15,15 +15,15 @@ We now accept versioned formulae as long as they [meet the requirements](Version
 Software that can upgrade itself does not integrate well with Homebrew's own
 upgrade functionality. The self-update functionality should be disabled (if possible without complicating the formula).
 
-### We don’t like install-scripts that download unversioned things
+### We don’t like install scripts that download unversioned things
 We don't like install scripts that are pulling from the `master` branch of Git repositories or unversioned, unchecksummed tarballs. These should use `resource` blocks with specific revisions or checksummed tarballs instead. Note that we now allow tools like `cargo`, `gem` and `pip` to download specifically versioned libraries during installation.
 
 ### We don’t like binary formulae
 Our policy is that formulae in the core tap
 ([homebrew/core](https://github.com/Homebrew/homebrew-core)) must be open-source
 and either built from source or produce cross-platform binaries (e.g. Java, Mono).
 Binary-only formulae should go to
-[Homebrew Cask](https://github.com/Homebrew/homebrew-cask).
+[homebrew/cask](https://github.com/Homebrew/homebrew-cask).
```

Comment:
I think `Homebrew Cask` is the proper name…

Reply:
@EricFromCanada Would you be interested in working on that integration at all?